go.mod

```diff
@@ -3,7 +3,11 @@ module github.com/jenkins-x/jx-cdevents-adapter
 go 1.19
 
 require (
-	github.com/gorilla/mux v1.8.0 // indirect
-	github.com/sirupsen/logrus v1.9.0 // indirect
+	github.com/gorilla/mux v1.8.0
```

Comment:
Why does it still have mux? Check if there are some files which are using mux? or may be u need to do `go mod tidy`

Reply:
Yes, I guess i forgot to do go mod tidy